### DISCUSSION TITLE
How to ignore the line break format

### DISCUSSION THREAD

**Discussion by hantsy** [ktlint/ktlint#2868] [Q&A] [answered]
I am using kotliner-gradle in my project, which depends on Klint. 

I am unsure from which version, the line breaks for every line are formatted(eg. convert Unix line break to Windows, etc.) when running `./gradlew formatKolin`. But there are no changes at all. Git will track the file status as modified in the local cache, but when I commit and push the changes from formatting, Git will report there are no changes to push.

**Comment by paul-dingemans** [comment, marked as answer]
Duplicate of #2747. Fixed in ktlint version `1.4.x`.
